src/odemis/driver/xt_client.py

```diff
@@ -164,7 +198,10 @@ def set_scanning_size(self, x, y):
         y: (float)
             size for y in meters.
         """
+        if y is None:
+            _, y = self.get_scanning_size()
         self.client.set_scanning_size(x, y)
+        return self.get_scanning_size()
```

Comment:
Similarly, here, just make that wrapper direct, and the caller has to take care of the doing the extra work.
That said, the `set_scanning_size()` of the server only takes one argument (`x`). The y is hard-coded.